packages/redux/src/entities/schemas/facet.ts

```diff
@@ -4,17 +4,25 @@ import {
   type FacetValue,
 } from '@farfetch/blackout-client';
 import { schema } from 'normalizr';
+import type { FacetGroupNormalized } from '../index.js';
 
 export const getId = (
-  { value, valueUpperBound, groupsOn }: FacetValue,
-  { key, type: parentType }: FacetGroup,
-) =>
-  `${key}_${value}${
+  entity: FacetValue,
+  parent: FacetGroup | FacetGroupNormalized,
```

Comment:
`parent` will always be a `FacetGroup` with a `hash` property (if you do some changes that I will describe shortly) as `getId` is only called when applying normalisation to a facet so it should be this:

```suggestion
  parent: FacetGroupWithListingHash,
```

where `FacetGroupWithListingHash` is this:

```suggestion
export type FacetGroupWithListingHash = FacetGroup & { hash?: string };
```

Define that type in this file as the suggestion as it will be useful in `productsList` schema.

Reply:
Great, updated!